src/renderer.test.tsx

```diff
@@ -0,0 +1,447 @@
+import { renderAll, renderAllWithoutStyles, renderStandfirstText, renderText, transformHref } from 'renderer';
```

Comment:
Is this a new test file? I thought we already had a `renderer.test.tsx` file?

Reply:
I think this may be some changes committed accidentally from #795?

Reply:
yes thats right, bad merge, renderer.test.tsx changed back to .ts